core/experiments/wp-cli.php

```diff
@@ -78,18 +81,21 @@ public function deactivate( $args, $assoc_args ) {
 
 		// Activate experiment callback.
 		$activate = function ( $site = '' ) use ( $args, $is_network ) {
-			$success = 'Experiment deactivated successfully';
-			$error = 'Cannot deactivate experiment';
+			$experiments = explode( ',', $args[0] );
```

Comment:
Maybe extract a function that in shared for both `activate` and `deactivate` commands